tests/crd-controller/FlowTest.cs

```diff
@@ -155,7 +191,24 @@ private async Task DeployController()
             Console.WriteLine("Deploying CRD");
             
             RunKubectlCommand("apply -f deployment.yaml");
-            RunKubectlCommand("apply -f crd.yaml");
+
+            Console.WriteLine("running ugly patch");
```

Comment:
Can you add a comment that explains why this patch is required?

Reply:
I think it was because `preserveUnknownFields` wasn't supported on k8s version bellow 1.15, but required on 1.15 if a schema is specified. I'll try to remove it and see if we need it anymore.

Reply:
I think it was because `preserveUnknownFields` wasn't supported on k8s version bellow 1.15, but required on 1.15 if a schema is specified. I'll try to remove it and see if we need it anymore.

Reply:
So... look like this is required - restoring it and adding a comment